src/Faker/Provider/pt_BR/Company.php

```diff
@@ -14,6 +14,30 @@ class Company extends \Faker\Provider\Company
         '{{lastName}} Comercial Ltda.'
     );
 
+    /**
+     * @link https://www.mundovestibular.com.br/articles/15937/1/Os-50-melhores-empregos-e-os-50-piores/Paacutegina1.html
```

Comment:
Unfortunately the content of this page is copyrighted, meaning you can't copy it and publish it under a MIT license. 

I'm sorry, but I can't accept this PR for legal reasons.